jobsdb/jobsdb.go

```diff
@@ -1615,26 +1627,67 @@ func (jd *HandleT) invalidateCache(ds dataSetT) {
 	}
 }
 
-//Rename a dataset
-func (jd *HandleT) renameDS(ds dataSetT, allowMissing bool) {
+//mustRenameDS renames a dataset
+func (jd *HandleT) mustRenameDS(ds dataSetT) {
```

Comment:
```suggestion
func (jd *HandleT) mustRenameDS(ds dataSetT) error {
```

I would suggest starting returning errors in newly introduce methods

Reply:
ok, but still the caller will panic on error, at least for the time being